src/diffusion_for_multi_scale_molecular_dynamics/models/egnn.py

```diff
@@ -283,6 +286,7 @@ def __init__(
         coords_agg: str = "mean",
         message_agg: str = "mean",
         n_layers: int = 4,
+        num_classes: int = 2,
```

Comment:
I don't think we should have a default here. Force the user to specify the number of classes.